discussions/utils.py

```diff
@@ -60,3 +63,29 @@ def get_token_for_request(request, force_create=False):
         str: the token or None
     """
     return get_token_for_user(request.user, force_create=force_create)
+
+
+def email_mm_admin_about_stale_memberships(membership_ids):
+    """
+    Email list of stale memberships to mm admin
+
+    Args:
+        membership_ids(List): List of ids
+    """
+    if getattr(settings, 'ADMIN_EMAIL', None) is None:
+        raise ImproperlyConfigured('Setting ADMIN_EMAIL is not set')
+
+    subject = "List of stale memberships"
+    body = (
+        "Hi,\n"
+        "The following memberships were stale in the PercolateQueryMembership:\n\n"
+        "{membership_ids}"
```

Comment:
This list could be large and I don't think the specific ids are useful. I think it's probably more useful to inform about the scope of the issue, which would probably be a per-channel breakdown of how many recipients haven't been synced.

Reply:
is there any field like channel `name/title` ?

Reply:
`Channel` has an FK to `PercolateQuery`: https://github.com/mitodl/micromasters/blob/master/discussions/models.py#L29